package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "powerbi-client",
-  "version": "2.11.0",
+  "version": "2.12.2",
```

Comment:
Not sure if we want this to be 2.12.2, the version is should be 2.12.1

Reply:
We are releasing 2.12.2.